apps/frontend/src/styles/HomePage.css

```
@@ -0,0 +1,119 @@
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.hero-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+  padding: 12px 24px;
+  border-radius: 10px;
+  text-decoration: none;
```

Stylelint (17.4.0)

[error] Unexpected duplicate "text-decoration" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)